src/app/modules/main/chat_section/chat_content/messages/view.nim

```diff
@@ -17,6 +17,7 @@ QtObject:
       chatIcon: string
       chatType: int
       loading: bool
+      keepUnread: bool
```

Comment:
IMO `forceUnread` would be better, but it's on your choice 🙂

Reply:
Thanks for the proposal, I believe `keepUnread` is more inclusive, what do you think?